seurat-find-markers.R

```diff
@@ -119,17 +119,15 @@ suppressPackageStartupMessages(require(Seurat))
 seurat_object <- readRDS(opt$input_object_file)
 
 # Get results matrix
-
-results_matrix <- FindAllMarkers(
+results_matrix<-FindAllMarkers(
```

Comment:
We're missing a few parameters from this function (assay, slot, random.seed, latent.vars, pseudocount.use, return.thresh), maybe new ones or maybe I didn't put them in the v2 wrappers.

Reply:
Should we add them?

Reply:
Please, feel free to add things (or use the GitHub suggest mode).

Reply:
I have added most of them. I'm normally skipping the assay and random.seed, but maybe we should add them (in some places it is hard coded for the `RNA` assay, so that it is why I have been avoiding assay). slot, latent vars and pseudocount use, on the defaults I found or with NULL, for some reason, always left me with 0 marker genes for the examples I tried.